internal/internal.go

```diff
@@ -394,19 +394,21 @@ func (p *envoyExtAuthzGrpcServer) check(ctx context.Context, req interface{}) (*
 		return nil, stop, err
 	}
 
-	inputValue, err := ast.InterfaceToValue(input)
+	var inputValue ast.Value
+	inputValue, err = ast.InterfaceToValue(input)
```

Comment:
These changes look OK, but it took me quite a moment to understand why it matters to do it _like this_. I wonder if we'd make this more robust (against future changes, too) if we changed `stop()` to be `stop(err error)` instead of relying on the single `err` reference not being shadowed.

Reply:
That's a good idea. I'll look into that.